wgpu/examples/hello-triangle-em/main.rs

```diff
@@ -0,0 +1,134 @@
+use env_logger::Builder;
```

Comment:
does this need to be a separate example? Can we instead just have `#[cfg(target_os = "emscripten")]` within the original hello-triangle?